<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Second Page</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Vazir</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ترب سیاه</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="back_to_main_page_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
    </font>
   </property>
   <property name="text">
    <string>بازگشت به صفحه اصلی</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="digi_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>368</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vazir</family>
    </font>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>نام محصول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>قیمت محصول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>لینک</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="divar_table">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>368</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vazir</family>
    </font>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>نام محصول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>قیمت محصول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>لینک</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="sheipur_table">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>160</y>
     <width>368</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vazir</family>
    </font>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>نام محصول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>قیمت محصول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>لینک</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="digikala">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vazir</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>دیجی کالا</string>
   </property>
  </widget>
  <widget class="QLabel" name="divar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vazir</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>دیوار</string>
   </property>
  </widget>
  <widget class="QLabel" name="sheipur">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Vazir</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>شیپور</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>68</y>
     <width>1120</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(214, 214, 214)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>71</y>
     <width>295</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="comparison_button">
      <property name="font">
       <font>
        <family>Georgia</family>
       </font>
      </property>
      <property name="text">
       <string>مقایسه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="favorites_button">
      <property name="font">
       <font>
        <family>Georgia</family>
       </font>
      </property>
      <property name="text">
       <string>علاقه مندی ها</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>widget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>back_to_main_page_button</zorder>
  <zorder>groupBox</zorder>
  <zorder>digi_table</zorder>
  <zorder>divar_table</zorder>
  <zorder>sheipur_table</zorder>
  <zorder>digikala</zorder>
  <zorder>divar</zorder>
  <zorder>sheipur</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
